### DISCUSSION TITLE
Server Versions Helm Chart

### DISCUSSION THREAD

**Discussion by kmentch** [Unleash/unleash#4497] [Q&A] [answered]
I am wondering about using a different version of the server docker image when installing through Helm. The most recent helm chart is only up to date with server version 5.1.8 and I would like to upgrade to use 5.3.4, is it as simple as just setting image.tag in the values.yaml? If so is there any concern for things not working?

**Comment by Benmidi** [comment]
+1 could really use this bump too

**Reply by gardleopard** [reply, marked as answer]
Hi, sorry for the slow feedback loop on this. We have bumped unleash to version 5.4.4 here: https://github.com/Unleash/helm-charts/pull/92

It is possible to override the unleash version by overriding image.tag in values.yaml as you suggest, for example: `helm upgrade unleash unleash/unleash --set image.tag=5.4.3`

As long as you only upgrade minor versions of unleash, the chart should work fine. Also because we are using k8s deployments, its safe even if it fails to launch as the previous deployment wont be removed before the new one is healthy.
